lib/workers/repository/update/pr/changelog/github/index.ts

```diff
@@ -28,18 +28,21 @@ export async function getTags(
 ): Promise<string[]> {
   logger.trace('github.getTags()');
   try {
-    const url = `${endpoint}repos/${repository}/tags?per_page=100`;
-    const res = await http.getJson<GithubRestTag[]>(url, {
-      paginate: true,
-    });
-    const tags = res.body;
+    const tags = await queryTags(
+      {
+        registryUrl: endpoint,
+        packageName: repository,
+      },
+      http
+    );
 
     // istanbul ignore if
     if (!tags.length) {
       logger.debug(`No Github tags found for repository:${repository}`);
     }
 
-    return tags.map((tag) => tag.name).filter(Boolean);
+    const versions = tags.map(({ version }) => version);
```

Comment:
can we remove the variable again?

Reply:
Yes I think, just wonder why we placed this at all?